src/store/store.ts

```diff
@@ -35,7 +35,8 @@ import {
   StateFilingRestorationIF,
   ValidationFlagsIF,
   ActionKvIF,
-  CourtOrderIF } from '@/interfaces/'
+  CourtOrderIF,
+  AccountInformationIF } from '@/interfaces/'
```

Comment:
Leo can you please put ActionKvIF, CourtOrderIF, and AccountInformationIF in alphabetical order here? It's a small thing but it helps with reading the code in the future. Thanks!

Reply:
Fixed, thanks Leo! Resolving.